lib/backend/model/rule.go

```diff
@@ -20,7 +20,8 @@ import (
 )
 
 type Rule struct {
-	Action string `json:"action,omitempty" validate:"backendaction"`
+	Action    string  `json:"action,omitempty" validate:"backendaction"`
+	LogPrefix *string `json:"log_prefix,omitempty" validate:"omitempty"`
```

Comment:
Is there a mechanism in the code to return an error if LogPrefix is specified but the action is not "log"?